Dockerfile

```diff
@@ -37,7 +35,7 @@ ENV SIA_MODULES gctwhr
 COPY --from=zip_downloader /sia/siac /sia/siad /usr/bin/
 COPY scripts/*.sh ./
 
-EXPOSE 9980 9981 9982
+EXPOSE 9980 9981 9982 9983 9984
 
 HEALTHCHECK --interval=10s CMD ["./healthcheck.sh"]
```

Comment:
Should this be removed?

Reply:
Also `LABEL autoheal...` above

Reply:
Hm, good catch. For some reason GoLand refuses to find that when searching in the project...